packages/react-components/react-overflow/src/useOverflowContainer.ts

```diff
@@ -126,3 +82,41 @@ export const updateVisibilityAttribute: OnUpdateItemVisibility = ({ item, visibl
     item.element.setAttribute(DATA_OVERFLOWING, '');
   }
 };
+
+const createReactBindings = (overflowManager: OverflowManager | null) => ({
```

Comment:
My attempt to try a binding technique that does not involve repeated construct/destruct of instances